driver/src/main/java/org/neo4j/GraphDatabase.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j;
+
+import java.net.URI;
+
+import org.neo4j.driver.Config;
+import org.neo4j.driver.Driver;
+
+/**
+ * A static class to create a {@link Driver driver}
+ * <p>
+ * An example:
+ * <pre>
+ * {@code
+ *
+ * // Create a driver with default configuration
+ * Driver driver = GraphDatabase.driver( "neo4j://localhost:7687" );
+ *
+ * // Establish a session with a Neo4j instance
+ * Session session = driver.session();
+ * ** Do some work with the database... **
+ *
+ * // Release all the resources
+ * driver.close();
+ * }
+ * </pre>
+ */
+public class GraphDatabase
```

Comment:
This will conflict with https://github.com/neo4j/neo4j/pull/4674 -- and I'm not sure it's the best place anyway. Why not use factory methods in `org.neo4j.Driver` ?